frontend/src/pages/overview/overview/ChainDetails.js

```diff
@@ -62,6 +62,9 @@ export const ChainDetails = ({ chainID, chainName, assetType }) => {
   const decimals =
     chainInfo?.network?.config?.currencies?.[0]?.coinDecimals || 1;
   const logoURL = chainInfo?.network?.logos?.menu;
+  const PRICE_CHANGE_PERCENTAGE = Number(
+    tokensPriceInfo[originMinimalDenom]?.info?.["usd_24h_change"] || "-1"
```

Comment:
```suggestion
    tokensPriceInfo[originMinimalDenom]?.info?.["usd_24h_change"] || "0"
```

Default should be zero